# Goal
geometry_msgs/PoseStamped target
string                    pointing_frame
duration                  min_duration
float64                   max_velocity
---
# Result
---
# Feedback
float64 pointing_position_error
float64 pointing_angle_error
